web/app/js/locales/es/messages.json

```
@@ -155,6 +155,7 @@
   "tableTitleResponseInit": "Respuesta Inicial",
   "tableTitleTCP": "TCP",
   "tableTitleTCPMetrics": "Métricas TCP",
+  "Pods under the resource {resource} in the {namespace} namespace are missing tap configurations (restart these pods to enable tap)":"A los pods debajo del recurso {resource} en el espacio de nombres {namespace} les faltan configuraciones de toque (reinicie estos pods para habilitar el toque)",
```

We're already using "Namespace" elsewhere in Spanish as well, because it's so prevalent and the literal translation might be confusing, IMO. (same thing regarding "tap").
```suggestion
  "Pods under the resource {resource} in the {namespace} namespace are missing tap configurations (restart these pods to enable tap)":"A los pods debajo del recurso {resource} en el namespace {namespace} les hace falta la configuración de tap (reinicie estos pods para habilitar tap)",
```

---

Great feedback! Thanks for the suggestion!